openeo-geotrellis/src/main/scala/org/openeo/geotrellis/OpenEOProcessScriptBuilder.scala

```diff
@@ -906,6 +907,20 @@ class OpenEOProcessScriptBuilder {
     bandFunction
   }
 
+  private def arrayApplyFunction(arguments: java.util.Map[String, Object]): OpenEOProcess = {
+    logger.warn("array_apply under development! Don't use yet")
+    val storedArgs = contextStack.head
+    val inputFunction = storedArgs("data")
+    val processFunction = storedArgs("process")
+
+    val bandFunction = (context: Map[String, Any]) => (tiles: Seq[Tile]) => {
+      val data: Seq[Tile] = evaluateToTiles(inputFunction, context, tiles)
+      val mappedValues: Seq[Tile] = evaluateToTiles(processFunction, context, data)
```

Comment:
not entirely sure, but here I would expect some kind of loop over data?
Unless if the loop is somehow part of processFunction already?
Of course, when thinking about it, there's quite a few functions that will anyway loop over the sequence and apply to each tile individually.

Reply:
I still need to check if it goes over every band and time correctly.  But for a single tile it works